docs/install.rst

```diff
@@ -27,21 +27,21 @@ Install the Package
 Follow this procedure to prepare the environment and install the |productName| package.
 Perform this procedure on every node in the federation.
 
-1. Install a Python\* \  3.6 (or higher) virtual environment. 
+1. Install a Python 3.8 (>=3.6, <3.9) virtual environment using venv.
    
- See the `Virtualenv installation guide <https://virtualenv.pypa.io/en/latest/installation.html>`_ for details.
+ See the `Venv installation guide <https://docs.python.org/3/library/venv.html>`_ for details.
 
 2. Create a new Virtualenv environment for the project.
 
    .. code-block:: console
 
-      python3 -m virtualenv env_name
+      python3 -m venv venv
 
 3. Activate the virtual environment.
 
    .. code-block:: console
 
-      source env_name/bin/activate
+      source venv/bin/activate
```

Comment:
It is better to leave the `env_name` parameter in my opinion

Reply:
I'm not sure. This command could be copied without any modifications. But if we change it to `env_name`, user should define his own name every time.